dash/utils/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
+  "compilerOptions": {
+    "moduleResolution": "Bundler"
```

Comment:
i have no idea why i needed this, spent way too long trying to figure it out, but for some reason adding some new tests in this package caused a typecheck error, something with vite/rollup, and this was the suggested workaround.  https://github.com/vitest-dev/vitest/issues/4567